nix/pkgs/default.nix

```diff
@@ -121,10 +121,17 @@ let
   # mean that e.g. we can't build the client dep updating
   # script on Darwin.
   easyPS = pkgs.callPackage (sources.easy-purescript-nix) { };
+
   # We pull out some packages from easyPS that are a pain to get otherwise.
   # In particular, we used to build purty ourselves, but now its build is a nightmare.
   # This does mean we can't as easily control the version we get, though.
-  inherit (easyPS) purty purs spago;
+  inherit (easyPS) purty purs psc-package spago;
+
+  # There is a spago2nix in easyPS, but it doesn't (currently) work. It doesn't
```

Comment:
Can we say why not and when/if we should go back to it?

Reply:
Good call. Updated with instructions. 👍